content/javascript/concepts/map/terms/delete/delete.md

```diff
@@ -0,0 +1,29 @@
+---
+Title: '.delete()'
+Description: 'Removes the entry associated with a given key from a Map object.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Map'
+  - 'Object'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+Removes the entry associated with a given key from a `Map` object.
+
+## Syntax
```

Comment:
Woop, don't forget to add the syntax here!

Reply:
Might I suggest:

```js
myMap.delete(key);
```

Reply:
or instead of `myMap`, `map` is sufficient. I will say that prefacing these with "my" is what I've seen in many other entries but it's up to you~

Reply:
I think we should stick with `map` for now because I also know other documentation sources use `myMap` and I think our entries should have some differentiation.

Reply:
Also, woops good catch! 😌 I had no idea I'd forgotten to add syntax! 😂